nowcasting_dataset/data_sources/pv/live.py

```diff
@@ -90,7 +110,16 @@ def get_pv_power_from_database(history_duration: timedelta) -> pd.DataFrame:
 
     pv_yields_df.columns = encode_label(pv_yields_df.columns, label="pvoutput")
 
-    # interpolate in between, maximum 30 mins
-    pv_yields_df.interpolate(limit=3, limit_area="inside", inplace=True)
+    # interpolate in between, maximum 'live_interpolate_minutes' mins
+    # note data is in 5 minutes chunks
+    pv_yields_df = empty_df.join(pv_yields_df)
```

Comment:
Instead of creating an `empty_df` and `join`ing it, would something like this be a valid alternative:

```python
# Append a row of NaNs at `now`:
pv_yields_df = pv_yields_df.append(pd.Series(name=now))

# Interpolate up until `now`:
pv_yields_df = pv_yields_df.resample("5T").interpolate()
```

Reply:
I like the idea, but i think its safer with the join. It covers things like
1. what if the dataframe has no values in it?
2. What if there are no values for the start time, we should then add one at the start then
3. What if the now is already in the df?
4. What if we dont want to interpolate the values for more than 1 hour?

Reply:
Im gona merge, and we can always come back to it